### PR TITLE
Mahitha | #5 | 1.Adds date_modified column to the tables which are ge…

### PR DESCRIPTION
**Pull request by Mahitha1467**
…tting info from obs table 2.Displays min date_created and obs_datetime and max date_modified of the reg tables in the rsp view

### CODE REVIEW

File: src/main/java/org/bahmni/mart/config/view/RegViewDefinition.java [hunk 1]
```diff
@@ -71,6 +74,12 @@ private String createCoalesceQueries(List<String> columnNames, List<String> tabl
     private String getCoalesceQuery(List<String> tableNames, String columnName) {
         List<String> columnNames = tableNames.stream()
                 .map(tableName -> String.format("%s.%s", tableName, columnName)).collect(Collectors.toList());
+
+        if (OBS_DATETIME.equals(columnName) || DATE_CREATED.equals(columnName)) {
```
Comment (sumanmaity112) [inline src/main/java/org/bahmni/mart/config/view/RegViewDefinition.java:78]: @Mahitha1467 Please simplify the `if-else`. You don't need a **else** block after **return** statement

File: src/main/java/org/bahmni/mart/exports/ObsRecordExtractorForTable.java [hunk 2]
```diff
@@ -93,25 +96,64 @@ private boolean isConstraintName(String conceptName, String tableColumnName) {
...
+        }
+    }
+
+    private String getDateModified(List<Obs> record) {
+        String dateModified = null;
+        String dateCreated = getDateCreated(record);
+        if (StringUtils.isEmpty(dateCreated)) {
+            return null;
+        }
+        for (Obs obs: record) {
```
Comment (sumanmaity112) [inline src/main/java/org/bahmni/mart/exports/ObsRecordExtractorForTable.java:113]: Pull this into a different method

**Review by sumanmaity112 — CHANGES_REQUESTED** [review]
Hi @Mahitha1467, I added some comments, please have a look

File: src/main/java/org/bahmni/mart/config/view/RegViewDefinition.java (hunk 1 above)
Comment (Mahitha1467) [reply src/main/java/org/bahmni/mart/config/view/RegViewDefinition.java:78]: @sumanmaity112 that is else-if and I need that condition

**Review by prasanna214 — APPROVED** [review]

**Pull request merged**